packages/compass-import-export/src/components/import-modal.tsx

```diff
@@ -203,6 +153,10 @@ function ImportModal({
     React
   );
 
+  const handleImportBtnClicked = useCallback(() => {
+    startImport();
+  }, [startImport]);
```

Comment:
Feel free to leave this, but as I pointed out in another PR for this modal, this effectively does nothing but adds a bunch of additional function calls without actually bringing any memoization benefits as there is only one dependency in this callback that is directly used inside